client/src/main/scala/com.gu.contentapi.client/model/Queries.scala

```diff
@@ -11,6 +11,15 @@ sealed trait ContentApiQuery {
   override def toString = {
     s"${getClass.getSimpleName}(/$pathSegment${QueryStringParams(parameters)})"
   }
+
+  def url(location: String, parameters: Map[String, String]): String = {
+    require(!location.contains('?'), "must not specify parameters in URL")
+
+    location + QueryStringParams(parameters)
+  }
+
+  def apply(targetUrl: String, customParameters: Map[String, String] = Map.empty): String =
```

Comment:
I think that we should rename this method. I would be surprised if I called `ContentQueryApi(something)` and got back a URL.

Reply:
Hmm, I disagree here. I think the framework is that of a REST service and in that context, queries are just URLs in disguise.